functions/Test-DbaBackupInformation.ps1

```diff
@@ -148,6 +148,34 @@ Function Test-DbaBackupInformation {
 						}
 					}
 				}
+				#Easier to do FileStream checks out of the loop:
+				if ('s' -in ($DbHistory | Select-Object -ExpandProperty filelist | Select-Object Type -Unique).Type) {
+					if ((Get-DbaSpConfigure -SqlInstance $RestoreInstance -ConfigName FilestreamAccessLevel).RunningValue -eq 0){
+						Write-Message -Level Warning -Message "Database $Database contains FileStream data, and FileStream is not enable on the destination server"
+						$VerificationErrors++
+					}
+
+					$ExistingFS = Get-DbaFileStreamFolder -SqlInstance $RestoreInstance
+					#$ExistingFS = ((Get-DbaDatabase -SqlInstance $RestoreInstance).FileGroups | ?{$_.FileGroupType -eq 'FileStreamDataFileGroup'}).Files.FileName
+					Foreach ($FileStreamFolder in ($DbHistory | Select-Object -ExpandProperty filelist | Where-Object {$_.Type -eq 's'} | Select-Object PhysicalName -unique).PhysicalName){
+						If ((Get-ChildItem $FileStreamFolder -ErrorAction SilentlyContinue).count -gt 0){
+							Write-Message -Level Warning -Message "Folder $FileStreamFolder already exists and contains data. Cannot use to restore $Database on $RestoreInstance"
+							$VerificationErrors++
+						}
+						if ($null -ne $ExistingFS){
+							if ($null -ne ($ExistingFs | Where-Object {$_.Database -eq $Database}) -and $Withreplace -ne $True){
+								Write-Message -Level Warning -Message "Folder $FileStreamFolder already in use for Filestream data on $RestoreInstance and WithReplace not specified, cannot restore"	
+								$VerificationErrors++
+							}
+							$OtherOwners = $ExistingFs | Where-Object {$_.FileStreamFolder -eq $FileStreamFolder -and $_.Database -ne $Database}
+							if ($null -ne $OtherOwners){
+								Write-Message -Level Warning -Message "Folder $FileStreamFolder already in use for Filestream data by $($OtherOwners.Database) on $RestoreInstance, cannot restore"
```

Comment:
For consistency, should be `$SqlInstance` instead of `$ResotreInstance`. Otherwise, the name come with brackets.

![image](https://user-images.githubusercontent.com/19521315/34162781-ce26f7b4-e4cc-11e7-9cf2-9ff988e1ca22.png)